src/libs/SidebarUtils.js

```diff
@@ -269,6 +269,10 @@ function getOptionData(reportID) {
     let lastMessageTextFromReport = '';
     if (ReportUtils.isReportMessageAttachment({text: report.lastMessageText, html: report.lastMessageHtml})) {
         lastMessageTextFromReport = `[${Localize.translateLocal('common.attachment')}]`;
+    } else if (ReportActionsUtils.isReportReviewAction(lastReportActions[report.reportID])) {
+        const iouReportID = ReportActionsUtils.getIouReportID(lastReportActions[report.reportID]);
+        const iouReport = ReportUtils.getReport(iouReportID);
```

Comment:
Make this inline since its not reused
```suggestion
        const iouReport = ReportUtils.getReport(ReportActionsUtils.getIouReportID(lastReportActions[report.reportID]));
```